packages/react-components/react-combobox/stories/src/Dropdown/DropdownAccessibility.md

```diff
@@ -17,5 +14,3 @@
 - **Don’t place the Dropdown button on a surface which doesn’t have a sufficient contrast.** The colors adjacent to the input should have a sufficient contrast. Particularly, the color of input with filled darker and lighter styles needs to provide greater than 3 to 1 contrast ratio against the immediate surrounding color to pass accessibility requirements. When using underline or outline styles, ensure the color of bottom border stroke has a sufficient contrast of greater than 3 to 1 against the immediate surrounding color.
```

Comment:
Interesting that we have an actual apostrophe here while most are single quotes.

`’` vs `'`

Don't really care one way or the other, just taking the opportunity to look for typos and grammar around the changes.

Reply:
oh, huh. Probably b/c I drafted it in a markdown editor and not in VS code

Reply:
actually -- looks like we have 35 results for `’` in `react-components/**/*.md`, so it's still largely biased towards the single quote, but mixed

Reply:
I think I'm probably going to leave it as-is, because trying to standardize `'` vs. `’` is probably a losing battle 😅